google/cloud/ndb/key.py

```diff
@@ -460,7 +466,7 @@ def __getnewargs__(self):
             state to pickle. The dictionary has three keys ``pairs``, ``app``
             and ``namespace``.
         """
-        return (
+        return (  # pragma: NO COVER
```

Comment:
Are we unable to test this for some reason?

Reply:
__getnewargs_ is never called in Python 2 code. Might change the NO COVER to NO PY2 COVER.